run-test.py

```diff
@@ -277,7 +319,7 @@ def k8s_start_pods(fname):
         data = get_output(k8s()+"get pods -l app=insolar-jepsen -o=json | "+\
             "jq -r '.items[].status.phase' | grep Running | wc -l")
         info("running pods: "+data)
-        if data == str(len(ALL_PODS)):
+        if data >= str(len(ALL_PODS)):
```

Comment:
This is weird. Why exactly we can get more pods than `len(ALL_PODS)`?

Reply:
Because we need at least len(ALL_PODS) running pods, there is no error if we get more than len(ALL_PODS). Now we have additional pod with tools and it is not in ALL_PODS